src/api/rootResolver.js

```diff
@@ -24,10 +56,39 @@ const resolvers = {
     },
     async parties() {
       return eventsList.map(event => ({ ...event, __typename: 'PartyMeta' }))
+    },
+    async events() {
```

Comment:
This can go in that eventsResolver. Maybe we should rename it something like 'logsResolver' as I was thinking of refactoring our internal names all to 'singleEvent', 'allEvents' for our parties, since we changed the name from blockparty

Reply:
If we are going to split off rootResolver, isn't it better to extract into the `deployerResolver` combining with `create` mutation function ? Otherwise we need to duplicate code to access to the `deployer` contract into two separate files.

Reply:
I am fine with this!